vector/src/main/res/values-v21/styles.xml

```diff
@@ -177,7 +177,19 @@
         <!-- android ui button -->
         <item name="android:colorAccent">?attr/editTextColor</item>
         <item name="colorAccent">?attr/editTextColor</item>
+    </style>
+
+
+    <style name="CountryPickerTheme" parent="AppTheme.NoActionBar">
```

Comment:
Check if it is useful to have that style in v21 file since it's same as base style